cidc_api/models/files/facets.py

```diff
@@ -46,44 +46,33 @@ def __init__(self, match_clauses: List[str], description: Optional[str] = None):
         ),
         "Cell Counts": FacetConfig(
             [
-                "/cytof_10021_analysis/cell_counts_assignment.csv",
-                "/cytof_10021_analysis/cell_counts_compartment.csv",
-                "/cytof_10021_analysis/cell_counts_profiling.csv",
-                "/cytof_e4412_analysis/cell_counts_assignment.csv",
-                "/cytof_e4412_analysis/cell_counts_compartment.csv",
-                "/cytof_e4412_analysis/cell_counts_profiling.csv",
+                "/cytof_analysis/cell_counts_assignment.csv",
+                "/cytof_nalysis/cell_counts_compartment.csv",
+                "/cytof_analysis/profiling.csv",
             ],
             "Summary cell count expression of individual cell types in each sample",
         ),
         "Combined Cell Counts": FacetConfig(
             [
-                "/cytof_analysis/combined_cell_counts_compartment.csv",
-                "/cytof_analysis/combined_cell_counts_assignment.csv",
-                "/cytof_analysis/combined_cell_counts_profiling.csv",
+                "/cytof_analysis/cell_counts_compartment.csv",
+                "/cytof_analysis/cell_counts_assignment.csv",
+                "/cytof_analysis/cell_counts_profiling.csv",
```

Comment:
I don't think this will work. If both sample-level cell counts and combined cell counts files have the same facet group, both file types will show up in the "Cell Counts" and "Combined Cell Counts" subfacets.

Reply:
(Specifically for facet groups `/cytof_analysis/cell_counts_compartment.csv` and `/cytof_analysis/cell_counts_assignment.csv`)

Reply:
in next commit